Source/Core/VideoCommon/VertexShaderGen.cpp

```diff
@@ -17,7 +17,6 @@
 VertexShaderUid GetVertexShaderUid()
 {
   ASSERT(bpmem.genMode.numtexgens == xfmem.numTexGen.numTexGens);
-  ASSERT(bpmem.genMode.numcolchans == xfmem.numChan.numColorChans);
```

Comment:
I'm not sure if this assert should be removed; based on 940bccd2412fdec5d22f53d9a9e28fec8ef41207 it was targeted at [bug 10180](https://bugs.dolphin-emu.org/issues/10180), which was instead fixed by #8717.

Reply:
I can confirm that the assert isn't hit anymore since #8717 skips rendering in this case.  So, it probably shouldn't be removed.

Reply:
Assert re-added.